test/SourceKit/Inputs/header_with_macro/Header.h

```diff
@@ -0,0 +1,6 @@
+#ifndef Header_h
```

Comment:
You can keep it in Inputs if you want, but I much prefer using `split-file`. I admit it's a little weird when cross language though.

Reply:
Hmm yeah, it being cross language was the main reason I didn't use `split-file`, didn't want to confuse the editor by putting C code in a swift file. That being said, I guess this is a simple enough case where it doesn't matter too much.